ui/components/DashboardComponent.js

```diff
@@ -81,7 +81,6 @@ const styles = (theme) => ({
   },
   redirectButton: {
     marginLeft: '-.5em',
-    backgroundColor: '#ccc',
```

Comment:
A slightly off color from the exist red notification will draw attention, but perhaps, avoid the jarring affect that this light gray induced.

What will the "Settings" link/button look like now?

Reply:
It will be in ' Black' color, and on hovering a bit of dark red background color will come up.